README.md

```diff
@@ -192,6 +196,11 @@ In order to use the API within `polkadot.js` you'll need to configure the follow
     "metadata_key": "TokenMetadataKey",
     "metadata_value": "TokenMetadataValue"
   },
+  "FixedOutputMetadataValueType": {
```

Comment:
should have restriction on the end? `FixedOutputMetadataValueTypeRestriction`

Reply:
renamed